iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/AbstractTransformerXml.java

```diff
@@ -20,6 +22,10 @@
  */
 abstract class AbstractTransformerXml<T extends Connectable, A extends IdentifiableAdder<A>> extends AbstractConnectableXml<T, A, Substation> {
 
+    private interface StepConsumer {
```

Comment:
Probably also static?

Reply:
Nested interfaces are apparently static by default (https://docs.oracle.com/javase/specs/jls/se8/html/jls-8.html#jls-8.5.1) so the modifier is redundant.

> A member interface is implicitly static (§9.1.1). It is permitted for the declaration of a member interface to redundantly specify the static modifier.